aiohttp_xmlrpc/handler.py

```diff
@@ -94,7 +92,7 @@ def _handle(self):
         else:
             kwargs = {}
 
-        result = yield from asyncio.coroutine(method)(*args, **kwargs)
+        result = await asyncio.coroutine(method)(*args, **kwargs)
```

Comment:
Oh woops this isn't right @mosquito

Reply:
Yes but we have to replace it like this https://github.com/mosquito/aiormq/blob/master/aiormq/tools.py#L21